src/components/dev-hub/star-rating.js

```diff
@@ -29,7 +29,7 @@ const StyledButton = styled(Button)`
     }
 `;
 
-const StyledContainer = styled('div')`
+export const StyledContainer = styled('div')`
```

Comment:
Using the `className` feature in the other comment we likely won't need to export these anymore